scheduler.go

```diff
@@ -210,23 +210,24 @@ func (s *Scheduler) removeByCondition(shouldRemove func(*Job) bool) {
 	}
 }
 
-// Remove First found Job by Tag
-func (s *Scheduler) RemoveJobByTag(tag string) {
-	jobindex := s.findJobsIndexByTag(tag)
+// RemoveJobByTag will Remove Jobs by Tag
+func (s *Scheduler) RemoveJobByTag(tag string) error {
+	jobindex, err := s.findJobsIndexByTag(tag)
 	// Remove Only if return valid index
-	if jobindex >= 0 {
+	if err == nil && jobindex >= 0 {
 		s.jobs = removeAtIndex(s.jobs, jobindex)
 	}
+	return err
```

Comment:
The standard "Golang" way would be this
```suggestion
	if err != nil {
		return err
	}
	s.jobs = removeAtIndex(s.jobs, jobindex)
```

Reply:
thank you, actually i use that logic, but idk why i so paranoid about what if somehow the return always remove at that index so i decide to move the logic like this. but i think it's just my overthinking, I am very okay with your suggestion. Thank you